run-fpm.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+if ! command -v mpifort > /dev/null 2>&1; then
+  brew install open-mpi
+fi
+
+fpm $@ \
+  --compiler `which mpifort` \
+  --c-compiler `which mpicc` \
+  --c-flag "-DPREFIX_NAME=_gfortran_caf_ -DGCC_GE_7 -DGCC_GE_8" \
+  --flag "-fcoarray=lib" \
+  --runner "mpirun -n 4"
```

Comment:
Is the idea that this is the fpm command to be used if one wants to make use of OpenCoarrays? If so, shouldn't there be a flag that points to the installed opencoarrays library?, or is the idea that it would be listed in the project's `fpm.toml` file and thus downloaded and compiled by fpm with those flags?